etc/polkit-rules-whitelist.json

```diff
@@ -84,5 +84,15 @@
 				}
 			}
 		}
+	},
+	"malcontent": {
+		"audits": {
+			"bsc#1177974": {
+				"comment": "Allows wheel members to bypass parental controls. We allow this as an exception (granting implicit authorization to wheel) since this is not security relevant per se.",
+				"digests": {
+					"/usr/share/polkit-1/rules.d/com.endlessm.ParentalControls.rules": "sha256:4dca105e78ff95c2317386d4df4f959f0c055eec13e12c34c48084b9bbb385b4"
```

Comment:
I thought files starting with a letter get ignored due to our `90-default-privs.rules`?

Reply:
True. At least when the action is also recorded in our polkit-default profiles, which is the case here.

I will ask the maintainer if they want to rename their file. I will submit the current state to Factory anyways so they can continue working on this.